provider/ec2/internal/ec2instancetypes/process_cost_data.go

```diff
@@ -169,7 +169,8 @@ func process(in io.Reader) (map[string][]instanceType, metadata, error) {
 	}
 	for _, sku := range skus.SortedValues() {
 		productInfo := index.Products[sku]
-		if productInfo.ProductFamily != "Compute Instance" {
+		if productInfo.ProductFamily != "Compute Instance" &&
+			productInfo.ProductFamily != "Compute Instance (bare metal)" {
```

Comment:
```suggestion
			if strings.HasPrefix(productInfo.ProductFamily) != "Compute Instance" {
```

Reply:
In general that is a good suggestion.  In this case, I think we want to know when these change to be on the lookout for potential issues moving forward.